cms/djangoapps/contentstore/tests/test_contentstore.py

```diff
@@ -1756,3 +1761,19 @@ def _create_course(test, course_key, course_data):
 def _get_course_id(course_data, key_class=SlashSeparatedCourseKey):
     """Returns the course ID (org/number/run)."""
     return key_class(course_data['org'], course_data['number'], course_data['run'])
+
+
+def _build_other_filter():
```

Comment:
Looks like a new one that I'll need to look at - a little odd. 
Any advice you can give on the thumbnail_location one would be appreciated? According to @dino-cikatic this is now gone, so no code path (on new systems, including test systems) will hit any line after ```if thumbnail_location``` - on systems with older datastores this line appears to be covered.